usd/src/sdf_parser/Material_Sdf2Usd_TEST.cc

```diff
@@ -0,0 +1,231 @@
+      }
+    }
+    else if (input.GetBaseName() == "diffuse_color_constant")
+    {
+      auto sourceInfoV = input.GetConnectedSources();
+      if (sourceInfoV.size() > 0)
+      {
+        pxr::UsdShadeInput connectedInput =
+          sourceInfoV[0].source.GetInput(sourceInfoV[0].sourceName);
+
+        const pxr::SdfPath& thisAttrPath = connectedInput.GetAttr().GetPath();
+        auto connectedPrim = _prim.GetStage()->GetPrimAtPath(
+          thisAttrPath.GetPrimPath());
+        if(connectedPrim)
+          connectedPrim.GetAttribute(
+            pxr::TfToken("inputs:diffuse_color_constant")).Get(&diffuseColor);
+      }
+      {
+        pxr::UsdShadeInput diffuseShaderInput =
+          variantshader.GetInput(pxr::TfToken("diffuse_color_constant"));
+        diffuseShaderInput.Get(&diffuseColor);
+      }
+      EXPECT_EQ(_diffuseColor, diffuseColor);
+    }
+  }
+}
+
+/////////////////////////////////////////////////
+// Fixture that creates a USD stage for each test case.
+class UsdStageFixture : public::testing::Test
+{
+  public: UsdStageFixture() = default;
+
+  protected: void SetUp() override
+  {
+    this->stage = pxr::UsdStage::CreateInMemory();
+    ASSERT_TRUE(this->stage);
+  }
+
+  public: pxr::UsdStageRefPtr stage;
+};
+
+/////////////////////////////////////////////////
+TEST_F(UsdStageFixture, Material)
+{
+  sdf::setFindCallback(sdf::usd::testing::findFileCb);
+  ignition::common::addFindFileURICallback(
+    std::bind(&sdf::usd::testing::FindResourceUri, std::placeholders::_1));
+
+  const auto path = sdf::testing::TestFile("sdf", "basic_shapes.sdf");
+  sdf::Root root;
+
+  ASSERT_TRUE(sdf::testing::LoadSdfFile(path, root));
+  ASSERT_EQ(1u, root.WorldCount());
+  auto world = root.WorldByIndex(0u);
+
+  const auto worldPath = std::string("/" + world->Name());
+  auto usdErrors = sdf::usd::ParseSdfWorld(*world, stage, worldPath);
+  EXPECT_TRUE(usdErrors.empty());
+
+  auto worldPrim = this->stage->GetPrimAtPath(pxr::SdfPath(worldPath));
+  ASSERT_TRUE(worldPrim);
+
+  std::string meshPath = worldPath + "/" + "mesh";
+  std::string meshLinkPath = meshPath + "/" + "link";
+  std::string meshVisualPath = meshLinkPath + "/" + "visual";
+  std::string meshGeometryPath =
+    meshVisualPath + "/" + "geometry";
+  auto meshGeometry = this->stage->GetPrimAtPath(
+    pxr::SdfPath(meshGeometryPath));
+  ASSERT_TRUE(meshGeometry);
+
+  {
+    std::string materialPath = "/Looks/Material_2";
+    auto materialPrim = this->stage->GetPrimAtPath(
+      pxr::SdfPath(materialPath));
+    EXPECT_TRUE(materialPrim);
+
+    std::string materialshaderPath = materialPath + "/Shader";
+    auto materialShaderPrim = this->stage->GetPrimAtPath(
+      pxr::SdfPath(materialshaderPath));
+    EXPECT_TRUE(materialShaderPrim);
+
+    CheckMaterial(materialShaderPrim,
+      pxr::GfVec3f(0.2, 0.5, 0.1),
+      pxr::GfVec3f(0, 0, 0),
+      "materials/textures/albedo_map.png",
+      "materials/textures/normal_map.png",
```

Comment:
When I run this test, I see error messages for these files:
```
[Err] [SystemPaths.cc:467] Could not resolve file [albedo_map.png]
[Err] [SystemPaths.cc:467] Could not resolve file [normal_map.png]
```

Does this need to be addressed?

Reply:
These failures are expected we are not including these files.